server/banner.ts

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs'
+import path from 'path'
+import logger from '../logger'
+import { BannerFile } from './@types/express'
+
+export default class Banner {
```

Comment:
Could you add some unit tests for this class?

Reply:
Added tests